### PR TITLE
Geolocation viewing

### PR DESCRIPTION
**Pull request by gillianpierce**
Hello Folks, 

Now we can see those fantastic locations as both an owner or borrower! Please review and merge into master 😋

-Gill

### CODE REVIEW

File: app/src/main/java/com/stl/skipthelibrary/Fragments/MapboxFragment.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        this.bookPickupLocation = new Location(latitude, longitude);
+        return inflater.inflate(R.layout.fragment_mapbox, parent, false);
+    }
+
+    // This event is triggered soon after onCreateView().
+    // Any view setup should occur here.  E.g., view lookups and attaching view listeners.
+    @Override
+    public void onViewCreated(View view, Bundle savedInstanceState) {
+        // Setup any handles to view objects here
+        // EditText etFoo = (EditText) view.findViewById(R.id.etFoo);
```
Comment (LukeSlev) [inline app/src/main/java/com/stl/skipthelibrary/Fragments/MapboxFragment.java]: leftovers¿

**Comment by LukeSlev** [comment]
Code seems fine, question is do we want the loco to stay on the book until the book is returned? That way we could say that the user and owner gotta meet there again to do the return. Or I guess they could figure that out on their own or something idk

File: app/src/main/res/layout/activity_map_box.xml [hunk 2]
```diff
@@ -23,17 +24,18 @@
...
         android:id="@+id/select_location_submit"
         android:layout_width="match_parent"
-        android:layout_marginRight="50dp"
+        android:layout_height="wrap_content"
         android:layout_marginLeft="50dp"
+        android:layout_marginRight="50dp"
         android:layout_marginBottom="25dp"
-        android:layout_height="wrap_content"
+        android:fontFamily="@font/poppins_semibold"
+        android:text="Submit Location"
```
[the remark this answers is not in the document]
Comment (gillianpierce) [reply app/src/main/res/layout/activity_map_box.xml]: Done, like dinner

File: app/src/main/java/com/stl/skipthelibrary/Fragments/MapboxFragment.java [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+import com.mapbox.mapboxsdk.geometry.LatLng;
+import com.mapbox.mapboxsdk.maps.MapView;
+import com.mapbox.mapboxsdk.maps.MapboxMap;
+import com.mapbox.mapboxsdk.maps.OnMapReadyCallback;
+import com.mapbox.mapboxsdk.maps.Style;
+import com.stl.skipthelibrary.Activities.MapBoxActivity;
+import com.stl.skipthelibrary.Entities.Location;
+import com.stl.skipthelibrary.R;
+
+import androidx.annotation.NonNull;
```
[the remark this answers is not in the document]
Comment (gillianpierce) [reply app/src/main/java/com/stl/skipthelibrary/Fragments/MapboxFragment.java]: Done, like breakfast

**Comment by gillianpierce** [comment]
> @gillianpierce minor things and it looks like users can move the pin on the map when they are viewing it, is it possible to lock the map view so you can't move it? Not the biggest deal as you could just come back to the screen to get original loco but ideally wouldn't have to do this

Fixed with most recent changes

**Pull request merged**